packages/interactivity-router/src/index.js

```diff
@@ -108,6 +116,7 @@ export const { state, actions } = store( 'core/router', {
 		 */
 		*navigate( href, options = {} ) {
 			const url = cleanUrl( href );
+			const { navigation } = state;
 			navigatingTo = href;
```

Comment:
Do we want to check `href` or `url`?

To be honest, `url` is a bad name for a `url` without hash. I know it's internal, but maybe we should make it more explicit and call it `urlWithoutHash`? 😄

Reply:
It's more like `pathWithQuery` because it doesn't include the host either, haha. 😅

As this is a way to reference a cached page, let me propose other names:

- `pageRef`: it indicates something that references a page, although it doesn't give any explicit clue of how its value is structured. Also, we're already using the term `ref` in other places, but with appending the term `page`, I don't see how it could create confusion.
- `pagePath` or simply `path`: that would denote clearly that it's not a complete URL. As the variable is only used internally, maybe it's not that important if we exclude the search part from the variable name, even though it's actually included in the value.
- `pathAndQuery`, `pathWithSearch` or any other combination where `path` and `query` or `search` are mentioned. That would explicitly represent the variable content but it actually tells nothing about its usage, which is to reference cached pages.

Reply:
Those are good. Let's go with `pagePath` for example 🙂

Reply:
Changed to `pagePath` in https://github.com/WordPress/gutenberg/pull/58377/commits/196634cfd50bfbdfd050b5cb168258ceaabe0792.